platform/app/public/config/default.js

```diff
@@ -52,6 +52,7 @@ window.config = {
         supportsFuzzyMatching: false,
         supportsWildcard: true,
         staticWado: true,
+        caseSensitive: false,
```

Comment:
Can you add it to the docs as well?

https://docs.ohif.org/configuration/configurationFiles

It would be useful to have this per query, so that the patient name search is case-insensitive, but other searches are not. Can you make this an object with keys?